Update the PulumiUP OpenGraph image (#1218)
pulumi/pulumi-hugo: themes/default/content/pulumi-up/_index.md

> title: PulumiUP

Reviewer: ```suggestion
title: PulumiUP: Virtual Cloud Engineering Event | May 4-5 2022
```

type: page
layout: pulumi-up-2022

> meta_image: /images/pulumiup/2022-save-the-date.png
> meta_desc: |
>     PulumiUP introduces you to the emerging practice of Cloud Engineering. Hear from the Pulumi team
>     and assorted industry experts.

Author: ```suggestion
meta_image: /images/pulumiup/2022-save-the-date.png
meta_desc: |
    DIFFERENT WORDS
```

> meta_desc: |
>     PulumiUP introduces you to the emerging practice of Cloud Engineering. Hear from the Pulumi team
>     and assorted industry experts.

Reviewer: ```suggestion
meta_desc: |
    Join us for PulumiUP 2022: free virtual conference on May 4th. Let's talk about the future of IaC, Cloud Engineering & #DevOps and Cloud. Hear from industry-recognized leaders, attend technical talks, demos, and panel discussions. #PulumiUP #infrastructureascode
```

Reviewer 2: @SaraDPH we have some validation that only allows meta descriptions to be at most 160 characters. We don't currently support separate text for social cards, but it is something we can implement!

Author: Definitely support adding this! Would be good to submit it as a follow-up PR, though -- I'll make an issue for that and add it to the list. (Should be a quick follow-up to this one.)
